proposals/null-coalescing-assignment.md

```diff
@@ -10,6 +10,8 @@
 
 Simplifies a common coding pattern where a variable is assigned a value if it is null.
 
+As part of this proposal, we will also loosen the type requirements on `??` to allow an expression whose type is the unconstrained type parameter to be used on the left-hand side.
```

Comment:
>the [](start = 113, length = 3)

"an expression whose type is **the** unconstrained type parameter" should be "an expression whose type is **an** unconstrained type parameter".